add darkmode

semanticdata opened the pull request:
Added dark-mode. This is related to #7. Followed the suggestion [here](https://github.com/zbrox/anpu-zola-theme/issues/7#issue-1662941398) to use [zola-hook](https://github.com/InputUsername/zola-hook)'s implementation.

semanticdata committed:
add darkmode

Implements dark-mode using the implementation from https://github.com/InputUsername/zola-hook. This is related to https://github.com/zbrox/anpu-zola-theme/issues/7.

zbrox commented:
Hey @semanticdata , thanks for the contribution!

I only have only two requests. Can you add attribution for the icons (with a url and make sure they're compatible to be distrubuted freely, the theme is under MIT)? And can you add another screenshot in the README showing off the dark theme?

semanticdata committed:
new darkmode icon, attributions, screenshots

semanticdata commented:
Good day @zbrox,

I have made the requested changes. Please review at your earliest convenience.

> Can you add attribution for the icons (with a URL and make sure they're compatible to be distributed freely, the theme is under MIT)?

I have sourced two new icons from [UXWing](https://uxwing.com/license/). A few things to highlight:

- All icons on UXWing free to download, and use personal, commercial projects.
- Attribution and Credit is NOT required, however, any credit will be much appreciated.
- There is no limit to the number of times you can use.
- You may modify the resources according to your requirements.
  
That said, I added a new Attributions and License sections to the README.

> Can you add another screenshot in the README showing off the dark theme?

I have updated the screenshot, and added a dark mode one as requested. I put them in a table, so they can look better while being side-by-side.

zbrox commented:
Thank you again for your work!
